scripts.js

```diff
@@ -0,0 +1,17 @@
+function Phone(brand, price, color) {
+    this.brand = brand;
+    this.price = price;
+    this.color = color;
+}
+
+Phone.prototype.printInfo = function() {
+    console.log("The phone brand is " + this.brand + ", color is " + this.color + " and the price is " + this.price + ".");
+}
+
+var iPhone6S = new Phone("Apple", 2250, "silver");
+var GalaxyS6 = new Phone("Samsung", 3000, "black");
```

Comment:
variable names should start with small letter - big letter at the beginning of names are reserved for constructor functions